marsappspa/src/App.tsx

```diff
@@ -9,7 +9,6 @@ import ComponentPage from "./ComponentPage";
 import CounterPage from "./CounterPage";
 import HomePage from "./HomePage";
 
-//
```

Comment:
let's change the last 2 routes to /component and /component/complex and use nested routes